sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/statistic/metric/occupy/OccupiableBucketLeapArray.java

```diff
@@ -98,4 +98,20 @@ public void debug(long time) {
         }
         System.out.println(sb.toString());
     }
+
+    /**
+     * just for test
+     * @param timeMillis
+     * @return
+     */
+    public long currentWaitingJustForTest(long timeMillis) {
```

Comment:
Adding a public method just for testing is not recommended. Maybe we can refer to #546 to fix the time-based unstable test cases. Use `AbstractTimeBasedTest` to adjust current time.

Reply:
OK, I have solved it.

Reply:
👍